reconcile/utils/terrascript_client.py

```diff
@@ -3155,6 +3166,145 @@ def is_elasticsearch_domain_name_valid(name):
         pattern = r'^[a-z][a-z0-9-]+$'
         return re.search(pattern, name)
 
+    @staticmethod
+    def elasticsearch_log_group_identifier(
+        domain_identifier: str,
+        log_type: ElasticSearchLogGroupType
+    ) -> str:
+        log_type_name = log_type.value.lower()
+        return f'OpenSearchService__{domain_identifier}__{log_type_name}'
+
+    def _elasticsearch_aggregate_log_groups_per_account(
+        self
+    ) -> Iterable[Tuple[str, str, str, str]]:
```

Comment:
Be concrete with the return types. In this case, `list[tuple[str, str, str, str]]` looks more appropriate.

Reply:
You are right. I misread our dev-guidelines. Abstract types should only be favored for arguments; not return types - makes more sense :+1: 

Fixed